blaze-bs4/blaze-bs4-sync.js

```diff
@@ -117,7 +117,7 @@ export const BlazeBs4 = {
   toast: {
     template: 'toasts',
     debug: false,
-    async load() {
+    load() {
       return import('./toast/toast.js');
```

Comment:
The import itself is still using the async version `import(...)` but in sync we need the static version `import '...'` because the compiler will resolve them to two total different constructs.

Reply:
I believe I fixed that in the latest commit